source/system/Memory.ooc

```diff
@@ -50,3 +52,6 @@ GlobalCleanup: class {
 		}
 	}
 }
+
+// Must be here because Mutex.ooc is loaded before Memory.ooc
```

Comment:
I don't know, the only problems we could have are in `system` where there's only `Mutex` and `String`, and only the first posed problems.